src/calibre/gui2/catalog/catalog_epub_mobi.py

```diff
@@ -806,7 +806,7 @@ def preset_save(self):
             self.preset_field.blockSignals(True)
             self.preset_field.clear()
             self.preset_field.addItem('')
-            self.preset_field_values = sorted([q for q in self.presets], key=sort_key)
+            self.preset_field_values = sorted((q for q in self.presets), key=sort_key)
```

Comment:
Is this loop needed at all? `sorted(self.presets, key=sort_key)` should be the same. (Above and below too.)

Reply:
I must have been tired when I hunted down these additional instances...